README.md

```diff
@@ -247,6 +247,7 @@ Focus is on the specific task.
 - [OpAuth](https://github.com/wouter0100/cakephp-opauth) - Opauth provides a standardized method for PHP applications to interface with authentication providers.
 - [Tools:Passwordable](https://github.com/dereuromark/cakephp-tools) - Containing [Passwordable behavior](https://github.com/dereuromark/cakephp-tools/blob/cake3/docs/Behavior/Passwordable.md) for a DRY approach on password hashing.
 - [TinyAuth plugin](https://github.com/dereuromark/cakephp-tinyauth) - Role based (single/multi) authentication as very light-weight approach.
+- [HyAuth plugin](https://github.com/btaens/hy-auth) - A CakePHP plugin for hierarchical, role based, simple authorization.
```

Comment:
Please respect alphabetical order :)
Also it would be good to have at least a basic test coverage (supported by travis CI).